src/Call.tsx

```diff
@@ -873,30 +873,29 @@ export class Call extends EventEmitter {
   /**
    * CallMessage API is in beta.
    *
-   * Send {@link (CallMessage:class)}.
```

Comment:
should we still link the class?

Reply:
Good catch, I would say we should. But I'll actually be refactoring the CallMessage classes/interfaces next sprint and this will change with that ticket, so I think it's OK to just leave it for now since I will get to it next sprint.